yt_dlp/extractor/bilibili.py

```diff
@@ -1025,6 +1060,64 @@ def _extract_video_metadata(self, url, video_id, season_id):
                 'description': self._html_search_meta('og:description', webpage)
             })
 
+    def _get_comments_reply(self, root_id, next_id=0, display_id=None):
+        comment_api_raw_data = self._download_json(
+            'https://api.bilibili.tv/reply/web/detail', display_id,
+            note=f'Downloading reply comment of {root_id}',
+            query={
+                'platform': 'web',
+                'ps': 3,  # comment's reply per page (default: 3)
+                'root': root_id,
+                'next': next_id,
+            })
+
+        for replies in traverse_obj(comment_api_raw_data, ('data', 'replies', ...)) or ():
+            yield {
+                'author': traverse_obj(replies, ('member', 'name')),
+                'author_id': traverse_obj(replies, ('member', 'mid')),
+                'author_thumbnail': traverse_obj(replies, ('member', 'face')),
+                'text': traverse_obj(replies, ('content', 'message')),
+                'id': replies.get('rpid'),
+                'like_count': int_or_none(replies.get('like_count')),
+                'parent': replies.get('parent'),
+                'timestamp': unified_timestamp(replies.get('ctime_text'))
+            }
+
+            if (not traverse_obj(comment_api_raw_data, ('data', 'cursor', 'is_end'))):
+                yield self._get_comments_reply(
```

Comment:
shouldn't this be `yield from`?

Reply:
Somehow it call the same `root_id` infinitely, so i'll keep `yield` for now

Reply:
Doesn't that return wrong structure through?

Reply:
Fixed in 3bef3ed8c3ae3eace035cda4390d896b5f040b3f